meta-dts-distro/recipes-dts/dts/files/ec_transition

```diff
@@ -82,6 +82,16 @@ board_config() {
           BIOS_LINK="https://cloud.3mdeb.com/index.php/s/SKpqSNzfFNY7AbK/download"
           EC_LINK="https://cloud.3mdeb.com/index.php/s/GK2KbXaYprkCCWM/download"
           ;;
+        "NS5x_NS7xPU")
+          HAVE_EC="true"
+          NEED_EC_RESET="true"
+          COMPATIBLE_EC_FW_VERSION="TBD"
```

Comment:
What are we going to do with those TBDs? @TomaszAIR @mkopec ?

Reply:
@mkopec @miczyg1 In other cases we used cloud links so if we have EC there, please provide links and EC_FW_VERSION then we could just add them here. Do we have tools to flash vendor EC? We would need to test this ec_transition on NS5x_NS7xPU